kie-remote/kie-remote-ws/kie-remote-ws-impl/pom.xml

```diff
@@ -70,4 +70,26 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <configuration>
+          <instructions>
+            <Bundle.Symbolic-Name>org.kie.remote.service.ws</Bundle.Symbolic-Name>
+            <Export-Package>
+              org.kie.remote.services.ws.*
+            </Export-Package>
+            <Import-Package>
+              *
+            </Import-Package>
+            <Bundle-Activator>org.drools.core.osgi.Activator</Bundle-Activator>
+            <_removeheaders>Private-Package</_removeheaders>
```

Comment:
I removed both the wrong activator and Private-Package configs and everything seems to work correctly).